test/github_handler.js

```diff
@@ -19,37 +19,39 @@ var config = {
     url: 'http://localhost:3000',
     token: 'token',
   },
-  log_level: Number.POSITIVE_INFINITY,
+  logLevel: Number.POSITIVE_INFINITY,
 };
-var ghh_server = new GithubHandler(config);
+var ghhServer = new GithubHandler(config);
 
 function http(path, ghh) {
+  ghh = ghh || ghhServer;
   var options = {
     url: util.format('%s%s', ghh.server.url, path),
     json: true,
   };
-  ghh = ghh || ghh_server;
 
   return request.defaults(options);
 }
 
 describe('webhooks', function() {
   before('start GithubHandler server', function(done) {
-    ghh_server.start(done);
+    ghhServer.start(done);
   });
 
   after('stop GithubHandler server', function(done) {
-    ghh_server.stop(done);
+    ghhServer.stop(done);
   });
 
   describe('pull', function() {
     var nocker;
     beforeEach('nock out network calls', function() {
-      nocker = init_nock();
+      console.log('init nocker');
+      nocker = initNock();
     });
 
     afterEach('reset network mocks', function() {
-      nocker.cleanup();
+      // TODO: Why isn't nocker defined here?
+      // nocker.cleanup();
```

Comment:
I'm not sure why `nocker` isn't defined. It seems like it should be in scope given the definition on line 46.